<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Acceso</class>
 <widget class="QWidget" name="Acceso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Acceso</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="lUsuario">
     <property name="text">
      <string>Usuario</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="leUsuario"/>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="leClave"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lClave">
     <property name="text">
      <string>Clave</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pbIngresar">
     <property name="text">
      <string>Ingresar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
